<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhoneDialerDialogClass</class>
 <widget class="QDialog" name="PhoneDialerDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>274</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wdtHistory" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tbwHistory">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CustomLabel" name="lblFullHistory">
        <property name="text">
         <string>Complete call history</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdtDialer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="lblCaption">
        <property name="text">
         <string>Calls</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lneNumber">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>Enter number</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
          <item>
           <widget class="CustomLabel" name="lblCity">
            <property name="text">
             <string notr="true">lblCity</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblCost">
            <property name="text">
             <string>lblCost</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gltButtons">
        <item row="0" column="0">
         <widget class="QPushButton" name="pbtNumber_1">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pbtNumber_2">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pbtNumber_3">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pbtNumber_4">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pbtNumber_5">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">5</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pbtNumber_6">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pbtNumber_7">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">7</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pbtNumber_8">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">8</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pbtNumber_9">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">9</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pbtNumber_10">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">*</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pbtNumber_11">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pbtNumber_12">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">#</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbtCall">
          <property name="text">
           <string>Call</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblBalance">
        <property name="text">
         <string notr="true">Balance</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header location="global">utils/customlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
